docs/UserGuide.adoc

```diff
@@ -1,4 +1,4 @@
-= AddressBook Level 3 - User Guide
+= HireLah! - User Guide
 :site-section: UserGuide
 :toc:
 :toc-title:
```

Comment:
change some of the metadata - repoURL, and the   
By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`